project_metadata.go

```diff
@@ -0,0 +1,16 @@
+package lifecycle
+
+const (
+	ProjectMetadataLabel    = "io.buildpacks.project"
+	ProjectMetadataFileName = "project-metadata.toml"
```

Comment:
The default value for this file should live in the `cmd` package with the other defaults